dev-lang/fpc/fpc-3.0.2-r1.ebuild

```diff
@@ -0,0 +1,166 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=5
+
+inherit toolchain-funcs
+
+HOMEPAGE="https://www.freepascal.org/"
+DESCRIPTION="Free Pascal Compiler"
+SRC_URI="mirror://sourceforge/freepascal/fpcbuild-${PV}.tar.gz
+	mirror://sourceforge/freepascal/fpc-${PV}.source.tar.gz
+	amd64? ( mirror://sourceforge/freepascal/${P}.x86_64-linux.tar )
+	x86? ( mirror://sourceforge/freepascal/${P}.i386-linux.tar )
+	doc? ( mirror://sourceforge/freepascal/Documentation/${PV}/doc-html.tar.gz -> ${P}-doc-html.tar.gz )"
+
+SLOT="0"
+LICENSE="GPL-2 LGPL-2.1-with-linking-exception"
+KEYWORDS="~amd64 ~arm ~ppc ~x86"
```

Comment:
No, and I have corrected the ebuild to exclude them. Still, I think it is very weird that repoman warned me about this - and specifically for this version and revision, not `=fpc-3.0.0`, where they were historically dropped:

```
dev-lang/fpc: removed arm and ppc keywords  
  
# Please enter the commit message for your changes.
# (Comment lines starting with '#' will not be included)
#
#  KEYWORDS.dropped              1
#   dev-lang/fpc/fpc-3.0.2-r1.ebuild: arm ppc
#  upstream.workaround           40
#   dev-lang/fpc/fpc-2.6.4.ebuild: Upstream parallel compilation bug (ebuild calls emake -j1 on line: 91)
#   dev-lang/fpc/fpc-2.6.4.ebuild: Upstream parallel compilation bug (ebuild calls emake -j1 on line: 97)
#   dev-lang/fpc/fpc-2.6.4.ebuild: Upstream parallel compilation bug (ebuild calls emake -j1 on line: 105)
#   dev-lang/fpc/fpc-2.6.4.ebuild: Upstream parallel compilation bug (ebuild calls emake -j1 on line: 107)
#   dev-lang/fpc/fpc-2.6.4.ebuild: Upstream parallel compilation bug (ebuild calls emake -j1 on line: 111)
#   dev-lang/fpc/fpc-2.6.4.ebuild: Upstream parallel compilation bug (ebuild calls emake -j1 on line: 125)
#   dev-lang/fpc/fpc-2.6.4.ebuild: Upstream parallel compilation bug (ebuild calls emake -j1 on line: 130)
#   dev-lang/fpc/fpc-2.6.4.ebuild: Upstream parallel compilation bug (ebuild calls emake -j1 on line: 133)
#   dev-lang/fpc/fpc-2.6.4.ebuild: Upstream parallel compilation bug (ebuild calls emake -j1 on line: 144)
#   dev-lang/fpc/fpc-2.6.4.ebuild: Upstream parallel compilation bug (ebuild calls emake -j1 on line: 150)
#   dev-lang/fpc/fpc-3.0.0.ebuild: Upstream parallel compilation bug (ebuild calls emake -j1 on line: 78)
#   dev-lang/fpc/fpc-3.0.0.ebuild: Upstream parallel compilation bug (ebuild calls emake -j1 on line: 84)
```